sdk/batch/batch-typespec/routes.tsp

```diff
@@ -2647,7 +2639,7 @@ applications can be returned.
   @query
   @minValue(1)
   @maxValue(1000)
-  
+  @projectedName("java", "maxPageSize")
   maxresults?: int32 = 1000;
```

Comment:
Treat `maxresults` as `maxpagesize`. Remove it from API.

User will call `pagedResponse.iterableByPage(preferredPageSize)`